denbi/perun/endpoint.py

```diff
@@ -169,8 +172,8 @@ def __import_scim_userdata__(self, json_obj):
         del_users = set(user_ids) ^ set(user_map.keys())
 
         for id in del_users:
-            self.logging.info("Deleting user %s", id)
             self.keystone.users_delete(id)
+            self.log2.info("Delete user [%s]", id)
```

Comment:
Not sure if deleting/creating should be on the update-domain. This however, is more a question of taste.

Reply:
"Update" is just the default name for the 2nd domain  and can be overwritten when using the library.